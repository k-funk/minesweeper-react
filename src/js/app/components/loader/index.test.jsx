import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import Loader from './index';


describe('outputs the expected tree when', () => {
  let wrapper;

  test('not loading', () => {
    wrapper = shallow((
      <Loader />
    ));
  });

  test('loading', () => {
    wrapper = shallow((
      <Loader loading />
    ));
  });

  afterEach(() => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
