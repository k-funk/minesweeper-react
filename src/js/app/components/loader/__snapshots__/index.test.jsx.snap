// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputs the expected tree when loading 1`] = `
<div
  className="spinner"
>
  <div
    className="bounce1"
  />
  <div
    className="bounce2"
  />
  <div
    className="bounce3"
  />
</div>
`;

exports[`outputs the expected tree when not loading 1`] = `""`;
